config/CLIConfiguration.ts

```diff
@@ -228,23 +235,50 @@ class _CLIConfiguration {
   }
 
   getDefaultAccount(): string | number | null {
-    return this.config && this.config.defaultAccount
-      ? this.config.defaultAccount
-      : null;
+    return (
+      this.getResolvedDefaultAccountForCWD() ||
+      this.config?.defaultAccount ||
+      null
+    );
   }
 
-  // TODO a util that returns the account to use, respecting the values set in
-  // "defaultAccountOverrides"
-  // Example "defaultAccountOverrides":
-  //  - /src/brodgers/customer-project-1: customer-account1
-  //  - /src/brodgers/customer-project-2: customer-account2
-  // "/src/brodgers/customer-project-1" is the path to the project dir
-  // "customer-account1" is the name of the account to use as the default for the specified dir
-  // These defaults take precedence over the standard default account specified in the config
-  getResolvedDefaultAccountForCWD(
-    nameOrId: string | number
-  ): CLIAccount_NEW | null {
-    return this.getAccount(nameOrId);
+  getResolvedDefaultAccountForCWD(): number | null {
+    const defaultOverrideFile = findup([DEFAULT_OVERRIDE_FILE_NAME], {
+      cwd: getCwd(),
+    });
+    if (!defaultOverrideFile) {
+      return null;
+    }
+    const source = fs.readFileSync(defaultOverrideFile, 'utf8');
+    const accountId = Number(source);
+
+    if (isNaN(accountId)) {
+      logger.error(
+        i18n(
+          `${i18nKey}.getResolvedDefaultAccountForCWD.errors.invalidAccountId`
+        )
+      );
+      process.exit(1);
```

Comment:
I think we should replace the `process.exit` calls in this file with an exception, we don't want to assume that these are always critical errors and should give the consuming application the chance to make that decision.

Reply:
Addressed in https://github.com/HubSpot/hubspot-local-dev-lib/pull/228/commits/2132b2299444bdcd8c31c37b4f6bd53cf28295b4